pkgs/development/libraries/avro-c++/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, cmake, boost155, pythonPackages
+}:
+
+let version = "1.7.5"; in
```

Comment:
No need to update this, but the idiom is
```nix
stdenv.mkDerivation rec {
  name = "foo-${version}";
  version = "4.2";
}
```
Note the `rec` keyword allows you to refer to keys within the same attrset.